src/store/modules/TsModsModule.ts

```diff
@@ -148,16 +145,95 @@ export const TsModsModule = {
         setExclusions(state, payload: string[]) {
             state.exclusions = payload;
         },
-        startBackgroundUpdate(state) {
-            state.isBackgroundUpdateInProgress = true;
+        setThunderstoreModListUpdateError(state, error: string|unknown) {
+            if (typeof error === 'string') {
+                state.thunderstoreModListUpdateError = error;
```

Comment:
Correct me if I'm wrong but this block should never be hit.

We invoke this in one place, and that's part of a try/catch block. We should always hit the `instanceof Error` case.